src/main/java/de/neuefische/OrderRepo/Order.java

```diff
@@ -0,0 +1,50 @@
+package de.neuefische.OrderRepo;
+
+import de.neuefische.Model.Product;
+import de.neuefische.ProductRepo.ProductDB;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Order {
+
+    protected int id;
+
+    protected ArrayList<Product> order = new ArrayList<>();
```

Comment:
Prinzip das du eine List von Produkten hast, ist richtig!
Besser wäre wenn du das Attribut `products` nennst.
Eine `Order` selbst erstellst du durch `new Order` (und in der Order hat man dann eine List von Produkten die bestellt wurden)

Reply:
Auch wenn es eine List of Orders sein soll ?

Reply:
Tatsächlich, ja!
Die Sache ist, dass diese Klasse `Order` die Bestellungen repräsentiert.
D.h. in dem `OrderRepository` verwaltet man dann die Liste an Bestellungen die da `List<Order> oders` ist

![image](https://user-images.githubusercontent.com/23424538/190346180-0037f3b3-da24-4904-baba-5fd13d31d737.png)